src/common/components/cards/rich-resolution-content.tsx

```diff
@@ -318,6 +318,15 @@ export const RichResolutionContent = NamedFC<RichResolutionContentProps>(
                     </span>
                 );
             }
+            case 'web/p-as-heading': {
+                return (
+                    <span>
+                        Inspect the {`<p>`} element in the Accessibility pane in the browser
+                        Developer tools and verify that the element is not used as a heading that is
```

Comment:
Do we own this text? The wording here is confusing. We want to move people towards using an actual heading, as opposed to simulating the visual appearance of a heading. Maybe something like "Inspect the <p> element to ensure that it uses an available heading level, rather than attempting to simulate the visual styling of the intended heading level"

Reply:
We do, it's PM provided! @nang4ally, thoughts about Dave's suggestion?

Reply:
Yeah, I like that modification @DaveTryon ! We can say "Inspect the <p> element and verify that the element is not used as a heading through visual styling with bold, italic text and font-size. If Headings are needed, use the appropriate heading tags"